main.go

```diff
@@ -0,0 +1,33 @@
+// Code generated by "go.opentelemetry.io/collector/cmd/builder". DO NOT EDIT.
```

Comment:
I'd suggest to add this in `cmd/clio` -- to avoid a top-level `main`, this allows us to more easily add multiple binaries per repo.